src/components/Admin/Tag.jsx

```diff
@@ -1,13 +1,13 @@
 import { colors } from "@/data/TagColors";
 
-const Tag = ({ color = "gray", text, onClick = null, classes }) => {
```

Comment:
make sure that all the tags r working fine despite this, many places depended on the default color

Reply:
remove the `= null` as well

Reply:
I am not sure what will be a working tag should be function as, but so far I checked that tags across working main pages seems to interact normally.